ste/mgr-JobPartTransferMgr.go

```diff
@@ -382,6 +383,13 @@ func (jptm *jobPartTransferMgr) Info() *TransferInfo {
 	}
 	blockSize = common.Iff(blockSize > common.MaxBlockBlobBlockSize, common.MaxBlockBlobBlockSize, blockSize)
 
+	// If the putBlobSize is 0, then the user didn't provide any putBlobSize, default to DefaultPutBlobSize
+	putBlobSize := dstBlobData.PutBlobSize
+	if putBlobSize == 0 {
+		putBlobSize = blockSize
+	}
+	putBlobSize = common.Iff(putBlobSize > common.MaxPutBlobSize, common.MaxPutBlobSize, putBlobSize)
```

Comment:
Instead of doing this during transfer, we can do this in init phase itself. If user has provided a putBlobSize bigger than MaxPutBlobSize then reset it to MaxPutBlobSize. Here we can have just one comparsion that if its zero then set it to blockSize.

Reply:
Its easier to do in ste since this input is taken in in many different commands. This is similar to how we do block size validation as well

Reply:
Didn't see this prior to my last comment. Does it really make sense to override the user's input here instead of indicating it is nonsensical? If we do override the user's input, we should probably log that we did, and make it generally apparent that we are doing something intentionally different, otherwise it may be looked at as a bug.

Reply:
Secondarily, DefaultPutBlobSize is unused.